dartsim/src/EntityManagementFeatures.cc

```diff
@@ -267,6 +267,64 @@ Identity EntityManagementFeatures::GetWorldOfModel(
   }
 }
 
+/////////////////////////////////////////////////
+std::size_t EntityManagementFeatures::GetNestedModelCount(
+    const Identity &_modelID) const
+{
+  return this->ReferenceInterface<ModelInfo>(_modelID)->nestedModels.size();
+}
+
+/////////////////////////////////////////////////
+Identity EntityManagementFeatures::GetNestedModel(
+    const Identity &_modelID, const std::size_t _modelIndex) const
+{
+  auto modelInfo = this->ReferenceInterface<ModelInfo>(_modelID);
```

Comment:
I believe that `modelInfo` can be marked `const` here.

Reply:
Made it `const` in https://github.com/ignitionrobotics/ign-physics/pull/228/commits/b836a0aa0fc7e9407e9f9d569491d3b9d32d9f82